UnityProject/Assets/Scripts/Lighting/LightSource.cs

```diff
@@ -50,7 +50,8 @@ private void Awake()
 		wallMount = GetComponent<LightMountStates>();
 		emergencyLightAnimator = GetComponent<EmergencyLightAnimator>();
 		SwitchState = true;
-		mState = InitialState;
+		if(isServer)
```

Comment:
IsServer won't be set in Awake. Override the OnStartServer NetworkBehaviour lifecycle method and set the mState value there

Reply:
You're right as the object itself wouldn't be ready on Awake. Did what you suggested.